improver_tests/utilities/test_TemporalInterpolation.py

```diff
@@ -676,6 +676,66 @@ def test_return_type(self):
         )
         self.assertIsInstance(result, iris.cube.CubeList)
 
+    def test_wind_direction_interpolation_over_north(self):
+        """Test that wind directions are interpolated properly at the 0/360
+        circular cross-over."""
+
+        data_time_0 = np.ones((self.npoints, self.npoints), dtype=np.float32) * 350
+        data_time_1 = np.ones((self.npoints, self.npoints), dtype=np.float32) * 20
+        domain_corner, grid_spacing = _grid_params("latlon", self.npoints)
+        cube_time_0 = set_up_variable_cube(
+            data_time_0,
+            units="degrees",
+            time=self.time_0,
+            frt=self.time_0,
+            domain_corner=domain_corner,
+            grid_spacing=grid_spacing,
+        )
+        cube_time_1 = set_up_variable_cube(
+            data_time_1,
+            units="degrees",
+            time=self.time_1,
+            frt=self.time_1,
+            domain_corner=domain_corner,
+            grid_spacing=grid_spacing,
+        )
+        expected_data = np.ones((self.npoints, self.npoints), dtype=np.float32) * 5
+        (result,) = TemporalInterpolation(interval_in_minutes=180).process(
+            cube_time_0, cube_time_1
+        )
+
+        self.assertArrayAlmostEqual(expected_data, result.data, decimal=4)
+
+    def test_wind_direction_interpolation(self):
```

Comment:
I tried using pytest.mark.parametrize, but couldn't get it to work, even when following the examples in the documentation. I have no idea what I'm doing wrong. So if someone else knows how to fix this easily, help would be appreciated. Otherwise, we could leave the two tests as they are rather than merging them into one test.

Reply:
Ah I see now that this file is using unittest not pytest. I don't think unittest has this functionality. Nevermind.